book/models/probabilistic_ode_solver_parameter_estimation.md

```diff
@@ -0,0 +1,364 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+file_format: mystnb
+mystnb:
+  execution_timeout: 200
+---
+
+<!-- #region -->
+# Parameter estimation in ODE models with a probabilistic ODE solver
+
+
+This tutorial explains how to estimate unknown parameters of initial value problems (IVPs) using 
+probabilistic solvers as provided by [ProbDiffEq](https://pnkraemer.github.io/probdiffeq/) in combination
+with Markov Chain Monte Carlo (MCMC) methods from [BlackJAX](https://blackjax-devs.github.io/blackjax/).
+
+## TL;DR
+Compute log-posterior of IVP parameters given observations of the IVP solution with ProbDiffEq. Sample from this posterior using BlackJAX.
+Evaluating the log-likelihood of the data is described [in this paper](https://arxiv.org/abs/2202.01287). 
```

Comment:
Just add an explicit citation to the paper with 
```
{cite:p}`tronarp2022fenrir`
```

you can leave `[in this paper](https://arxiv.org/abs/2202.01287)` if you want and just put the citation directly afterwards.